src/native-common/Image.tsx

```diff
@@ -79,12 +79,12 @@ export class Image extends React.Component<Types.ImageProps, Types.Stateless> im
 
     render() {
         // Check if require'd image resource
-        let imageSource: RN.ImageSource | number;
+        let imageSource: RN.ImageURISource | number;
```

Comment:
There is  type https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react-native/index.d.ts#L3402
```typescript
export type ImageSourcePropType = ImageURISource | ImageURISource[] | ImageRequireSource;
```

Reply:
Good point. Fixed.